template/context.go

```diff
@@ -27,7 +27,8 @@ type Context struct {
 	}
 
 	Node struct {
-		ID string
+		ID       string
+		Hostname string
```

Comment:
While we're at it, would it make sense to add other properties from `NodeDescription`, like `Architecture` and `OS`?